### DISCUSSION TITLE
Map starrate is shown as 0 stars in multiplayer

### DISCUSSION THREAD

**Discussion by Sinamer** [ppy/osu#15418] [Q&A] [answered]
In multiplayer, the beatmap card shows the difficulty as 0 stars:
![image](https://user-images.githubusercontent.com/53346335/139914374-654f91d3-93bc-4112-8e98-4dc508e85c99.png)

Here's a screenshot with the actual starrate of the map:
![image](https://user-images.githubusercontent.com/53346335/139914534-5f985c78-eb63-43e9-92be-9e6823a0fbf6.png)

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/15420
